common/include/vulkan_interfaces.h

```diff
@@ -23,6 +23,8 @@
 #ifdef DE_BUILD_VIDEO
 #include "vkDefs.hpp"
 using namespace vk;
+#elif defined(DE_BUILD_VVS)
+#include "vkDefs.h"
```

Comment:
AFAIK there's no need to add this since it's already defined just above, we just have to add the 

`add_compile_definition(DE_BUILD_VIDEO)` in CTS external/vulkan_video_samples